src/app/components/TradeTable/index.jsx

```diff
@@ -53,13 +54,28 @@ const TradeTable = ({ swaps, handleClick }) => (
   </Table>
 )
 
-const createStatusLabel = (status) => {
-  const icon = status == 'order_complete' ? (
-    <i style={{ fontSize: '18px' }} className='text-success fa fa-check-circle'></i>
-  ) : (
-    <i className='fa fa-spinner fa-pulse'/>
-  )
-  return <Expandable shrunk={icon} expanded={status.replace('_',' ')}></Expandable>
+//NOTE: if we end up not filtering out other statuses this approach wont work
+const statusInfo = {
```

Comment:
Need to add a status for `failed` orders that also tells them to contact support. Perhaps you could use `fa fa-exclamation-triangle text-danger`?